frontend/test/resolution/testDeprecationUnstable.cpp

```diff
@@ -275,8 +275,8 @@ static void test0(void) {
 
       var v1 = new r1();
       var v2 = new r2();
-      var v3 = new c1();
-      var v4 = new c2();
+      var v3 = new unmanaged c1();
+      var v4 = new unmanaged c2();
```

Comment:
What's not working with 'owned' here?

Reply:
nothing is not working, but we can avoid having to resolve parts of the `stdib` (and thus save some time) by using `unmanaegd`. I have done so where the use of stdlib features is not relevant to the test.